jumpscripts/healthchecks/dead_volumes_check_schedule.py

```diff
@@ -0,0 +1,46 @@
+from JumpScale import j
+
+descr = """
+Scheduler that runs on storagemaster to check the dead volumes
+"""
+
+organization = "greenitglobe"
+author = "ali.chaddad@gig.tech"
+version = "1.0"
+category = "monitor.healthcheck"
+
+enable = True
+async = True
+period = 21600  # 6 hrs
+roles = ["controller"]
+queue = "process"
+log = False
+timeout = 60
+
+
+def action():
+    acl = j.clients.agentcontroller.get()
+    results = []
+    job = acl.executeJumpscript(
+        "greenitglobe",
+        "dead_volumes_check",
+        role="storagemaster",
+        gid=j.application.whoAmI.gid,
+    )
+    if job["state"] == "OK":
+        results.extend(job["result"])
```

Comment:
what about if job not ok? Shoulnt you add a message that you failed to collect them?